cypress/integration/userbase.spec.js

```diff
@@ -0,0 +1,74 @@
+/// <reference types="Cypress" />
+
+describe('Configure the env', function () {
+  let userbase = {}
+  let info = {}
+  beforeEach(() => {
```

Comment:
I like the use of the beforeEach() to clean the slate before each unit test runs (as if it's a new session each time a unit test runs)

I'd suggest configuring the app Id here, rather than elsewhere. This way you won't need to wrap functions like sign up [in a command](https://github.com/encrypted-dev/userbase/pull/4/files#diff-9593faa52e480476d0f5eef99f197fbbR44-R47), or call configure before each function like with [sign in](https://github.com/encrypted-dev/userbase/pull/4/files#diff-c89a51dd36635d55404ef5f2b76c577dR49)

Reply:
I want to explicitly call configure before signin to allow me testing the error if a user calls signin without configure before it. That said I like your suggestion and will improve it a bit.